src/development/vortex/development/networks/models/registry.py

```diff
@@ -2,53 +2,14 @@
 from functools import partial
```

Comment:
I think this imports are no longer needed, so you can delete them

Reply:
okay